client-js/config.js

```diff
@@ -12,6 +12,7 @@ System.config({
   "map": {
     "d3": "github:mbostock/d3@3.5.6",
     "page": "npm:page@1.6.3",
+    "three": "github:mrdoob/three.js@master",
```

Comment:
Can we use a tagged version instead of master? This could pull down a slightly different version for different people depending on when they run jspm install.